.github/pull_request_template.md

```diff
@@ -14,8 +18,6 @@ If the pull request is a **bug-fix**, **enhancement** or a **refactor**, please
 
 - New capabilities/behavior
 
-If the pull request is a **new feature**,
-
 ### Any technical or performance related pointers to consider with the change?
```

Comment:
we need to keep relevant questions to make it user friendly. Can we move queries on 10 files & multiple linear tasks as Developer checklist instead ?
- Limited to 10 file changes in PR
- Limited to one linear task 

As these things shouldn't be the norm, we can have discussion as comments instead if this has to be exception. cc: @koladilip